src/memkind_memtier.c

```diff
@@ -98,6 +98,12 @@
 #define THRESHOLD_CHECK_CNT 20
 #define THRESHOLD_STEP      1024
 
+//PEBS
+    float old_time_window_hotness_weight;
+    float pebs_freq_hz;
+    unsigned long sample_frequency;
```

Comment:
This should be a float/double - we might want to test lower frequencies, e.g. 0.5 Hz or 0.1 Hz as well.

Please remember about adding some basic checks